.changeset/sour-frogs-shout.md

```diff
@@ -0,0 +1,25 @@
+---
+'astro': patch
+---
+
+Added support for optimizing remote images to `astro:assets`. This comes with two new parameters to specify which domains and host patterns are authorized for remote images.
+
+For example, the following configuration will only allow remote images from `astro.build` to be optimized:
+
+```ts
+export default defineConfig({
+  image: {
+    domains: ["astro.build/"],
+  }
+});
```

Comment:
```suggestion
// astro.config.mjs
export default defineConfig({
  image: {
    domains: ['astro.build'],
  }
});
```
Added the file name, and also changed the value here to match the docs example (though, THIS one may in fact be correct, in which case, change the other one so that they match). One had a trailing slash and one didn't, and even if they're both OK, we should show the same example in both places.